### PR TITLE
SOLENG - 8342 - Update go-device-logging.md to add Engine Hours Explainer

### PR DESCRIPTION
**Pull request by RichElliottGeoTab**
added blurb explaining DiagnosticEngineHoursId vs DiagnosticEngineHoursAdjustmentId

### CODE REVIEW

File: src/software/guides/go-device-logging.md [hunk 1]
```diff
@@ -123,3 +123,16 @@ Vehicles will not necessarily return all of the above information; the list is b
 Seat belt and odometer requests are proprietary on most passenger cars. It is Geotab's goal to support seat belt and odometer across all the major vehicle manufacturers. If you are unable to obtain seatbelt or odometer requests for your vehicle, please contact Geotab Support. You can query the percentage of seat belt, odometer and other engine based data is supported for different vehicle types via MyAdmin.
 
 Seat belt use is logged on status change: a value of 1 represents the seat belt unbuckled, while a value of 0 represents the seat belt buckled. Odometer is logged both on ignition on and ignition off and every hour.
+
+#### Engine Hours
+It is often important in fleet tracking to look at a vehicle's engine hours. While Geotab strives to support this data on as many vehicles as possible, it is sometimes not possible. This means it is important to distinguish between our two engine hours types of StatusData in order to utilize them appropriately.
```
Comment (muyuncaigeotab) [inline src/software/guides/go-device-logging.md:128]: Could you change this "This means it is important to distinguish between our two engine hours types of StatusData in order to utilize them appropriately." 
to
"As a workaround, Geotab provides two types of engine hours StatusData. It is important to distinguish between the two in order to utilize them appropriately."

File: src/software/guides/go-device-logging.md [hunk 2]
```diff
@@ -123,3 +123,16 @@ Vehicles will not necessarily return all of the above information; the list is b
...
+
+DiagnosticEngineHoursId:
+This is the raw engine hours provided by the ECU in the vehicle as read by the GoDevice.
+
+DiagnosticEngineHoursAdjustmentId:
+Records the GPS travel time for vehicles which do not report engine hours from their ECU. An engine hours offset can also be manually added which will then report as total GPS travel time plus the engine hours offset. When the manual engine hours entry is updated, the total GPS travel time used in the calculation is reset to zero before continuing to increment.
+
+Add<StatusData><DiagnosticEngineHoursAdjustmentId> is used to add manual entries (becomes the new engine hours offset).
+Get<StatusData><DIagnosticEngineHoursAdjustmentId> returns the engine hours as engine hours offset plus total GPS travel time since the offset was set.
```
Comment (muyuncaigeotab) [inline src/software/guides/go-device-logging.md:137]: In line 136 and 137, could you change the raw text from "Add\<StatusData\>\<DiagnosticEngineHoursAdjustmentId\>" to "Add\\\<StatusData\\\>\\\<DiagnosticEngineHoursAdjustmentId\\\>"? Without the escape character \ , the <> get interpreted as html tags, and get ignored. Compare raw text with the [rich diff](https://github.com/Geotab/sdk/pull/182/files?short_path=d1e7086#diff-d1e7086cf81f5d9b5b2611dbe2eaa8bfa8b2a85737d9033d95135c95983cf3f1) and you'll see what I meant.

**Review by muyuncaigeotab — CHANGES_REQUESTED** [review]

File: src/software/guides/go-device-logging.md (hunk 1 above)
Comment (Stephen-F) [reply src/software/guides/go-device-logging.md]: @RichElliottGeoTab ". it" Needs to be capitalized.

Comment (RichElliottGeoTab) [reply src/software/guides/go-device-logging.md]: @Stephen-F All fixed, should be good for merge

**Comment by Stephen-F** [comment]
QAed.

**Pull request merged**